main.tf

```diff
@@ -31,6 +31,8 @@ locals {
       message_retention_duration = lookup(subscription, "message_retention_duration", var.default_message_retention_duration)
       expiry_ttl                 = lookup(subscription, "expiry_ttl", local.default_expiry_ttl)
       filter                     = lookup(subscription, "filter", "")
+      minimum_backoff            = lookup(subscription, "minimum_backoff", var.default_minimum_backoff)
+      maximum_backoff            = lookup(subscription, "maximum_backoff", var.default_maximum_backoff)
```

Comment:
@koshmikumari we will need to add these backoff attributes for `local. push_subscriptions` as well